src/sentry/static/sentry/app/components/commitLink.jsx

```diff
@@ -38,16 +44,22 @@ class CommitLink extends React.Component {
         href={commitUrl}
         target="_blank"
       >
-        {repository.provider.id == 'github' && (
-          <InlineSvg src="icon-github" style={{verticalAlign: 'text-top'}} size="14px" />
-        )}
-        {repository.provider.id == 'bitbucket' && (
-          <InlineSvg
-            src="icon-bitbucket"
-            style={{verticalAlign: 'text-top'}}
-            size="14px"
-          />
-        )}
+        {repository.provider.id == 'github' ||
```

Comment:
If we're repeating this check all over might be worth having a `isBitbucket` helper method.

```javascript
const isBitbucket = (providerId) => ['bitbucket', 'bitbucket:integrations'].includes(providerId);
```